### ISSUE TITLE
Year 2038 problem

### ISSUE THREAD

**Issue by batphil** [siddii/angular-timer#151]
What about the 'Year 2038 problem' for the countdown ?

**Comment by siddii** [comment]
No, I haven't looked into that yet. Can you send a plunkr highlighting the issue?
